onward/app/controllers/MostPopularController.scala

```diff
@@ -20,11 +20,17 @@ object MostPopularController extends Controller with Logging with ExecutionConte
   def renderHtml(path: String) = render(path)
   def render(path: String) = Action.async { implicit request =>
     val edition = Edition(request)
-    val globalPopular = MostPopular("across the guardian", "", MostPopularAgent.mostPopular(edition))
+    val globalPopular: Option[MostPopular] = {
+      var globalPopularContent = MostPopularAgent.mostPopular(edition)
+      if (globalPopularContent.nonEmpty)
+        Some(MostPopular("across the guardian", "", globalPopularContent))
+      else
+        None
+    }
```

Comment:
I might have been tempted to make `MostPopularAgent.mostPopular` return an `Option` instead. That would keep working code cleaner and make all people who decide to use it understand that an empty Most Popular is different to a populated one.